specs/capri/FileShare_Design.md

```diff
@@ -383,7 +383,9 @@ This is the list of proposal for API
 
     “accessCapability” : "[]string"read/write
 
-    “accessTo” : "string"   //“The IP's”
+    “accessTo” : "[]string"   //“user_list”
+ 
+    "status"   : "string"   //track acl status
```

Comment:
if we are adding an ACL, for a list of users with a list of capabilities, status should also be a list ? what will status be if ACL can be done for one user and not for another ?